csrc/kernels.cu

```diff
@@ -627,7 +628,7 @@ __global__ void kQuantizeBlockwise(float * code, T * __restrict__ const A, float
     for(int i = threadIdx.x; i < 256; i+=blockDim.x)
       smem_code[i] = code[i];
 
-  for (unsigned int i = base_idx; i < n_full; i += gridDim.x*BLOCK_SIZE)
+  for (int i = base_idx; i < n_full; i += gridDim.x*BLOCK_SIZE)
```

Comment:
Why the type change?

Reply:
This is a super tiny optimization based on advice in the best practices guide here:

https://docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#loop-counters-signed-vs-unsigned

Eventually it might be the sort of thing we have linting rules for.

Reply:
Thanks, the doc link is exactly what I was hoping for, and explains why :)